racs_tools/beamcon_2D.py

```diff
@@ -227,7 +227,10 @@ def worker(
 
     datadict.update({"conbeam": conbeam, "final_beam": new_beam, "sfactor": sfactor})
     if not dryrun:
-        if (
+        if np.isnan(sfactor) or np.isnan(conbeam):
```

Comment:
@wasimraja81 Just a quick note that this fix catches a potential bug. In the 2D case if a cutoff is set in the current version an image with a PSF above this cutoff will cause a crash. This update fixes it